### PR TITLE
Fix/disabled behavior

### PR DESCRIPTION
**Pull request by lissavxo**
Related to #427 

Description
---
Fixed disabled behavior following [this](https://github.com/PayButton/paybutton/issues/427) instructions  

Test plan
---
Run the project with `yarn dev` or `yarn clean:build`
Using the generator
1 - Create a button active disabled option, the button text should be blurred 
2 - Create a widget active disabled option, the widget and button text should be blurred
3 - Create a widget active disabled option, add goal amount, the goal amount bar should be blurred too
4 - Test open dialog buttons in the `dev/demo/index.html` there are two buttons and one is disabled, the disabled button should not open the dialog and the text should change to unavailable when click

### CODE REVIEW

**Review by Klakurka — APPROVED** [review]
There's some "interesting" behavior depending on what settings you go with but this is fine IMO.

File: react/lib/components/Widget/Widget.tsx [hunk 1]
```diff
@@ -629,15 +629,17 @@ export const Widget: React.FunctionComponent<WidgetProps> = props => {
...
-            </Typography>
-          ) : (
-            <Typography className={classes.text}>
-              {loading ? 'Loading...' : success ? successText : text}
-            </Typography>
-          )}
+          <Typography className={errorMsg ? classes.error : classes.text}>
+          {errorMsg 
+            ? errorMsg 
+            : disabled
```
Comment (chedieck) [inline react/lib/components/Widget/Widget.tsx:635]: maybe some parenthesis and indentation here to make it more readable? The change makes the begginer more readable, but the part in the middle is quite hard to follow

**Review by chedieck — CHANGES_REQUESTED** [review]

**Review by Klakurka — DISMISSED** [review]

**Review by Klakurka — APPROVED** [review]

**Review by chedieck — APPROVED** [review]
Tested, the described behavior applies therefore I'm approving, but:

Why are we blurring the disabled button but not the dialog? I actually don't understand the purpose of the "open payment dialog" (which was already present on master), AFAICT it acts just like a PayButton.

**Review by chedieck — APPROVED** [review]
Test

**Review by chedieck — DISMISSED** [review]
IDK why this dismissal is happening, approving again

**Review by Klakurka — APPROVED** [review]
